ui/test/nuxeo-layout.test.js

```diff
@@ -27,57 +35,134 @@ const base = url.substring(0, url.lastIndexOf('/'));
 // Export Polymer and PolymerElement for 1.x and 2.x compat
 window.Polymer = Polymer;
 
-suite.skip('<nuxeo-layout>', () => {
-  test('layout not found', async () => {
-    const layout = await fixture(
-      html`
-        <nuxeo-layout href="notfound.html"></nuxeo-layout>
-      `,
-    );
-    await waitForEvent(layout, 'iron-resize');
-    expect(layout.$$('nuxeo-error').hidden).to.be.false;
-    expect(layout.element).to.be.undefined;
-  });
+suite('<nuxeo-layout>', () => {
+  suite('Error handling', () => {
+    test('Should display nuxeo-error when layout is not found', async () => {
+      const layout = await fixture(
+        html`
+          <nuxeo-layout href="notfound.html"></nuxeo-layout>
+        `,
+      );
+      await flush();
 
-  test('layout stamped with model', async () => {
-    const layout = await fixture(html`
-      <nuxeo-layout href="${base}/layouts/dummy-layout.html" model='{"text": "dummy"}'></nuxeo-layout>
-    `);
-    await waitChanged(layout, 'element');
-    expect(layout.$$('nuxeo-error').hidden).to.be.true;
-    expect(layout.element.localName).to.equal('dummy-layout');
-    expect(layout.element.text).to.equal('dummy');
-  });
+      const nuxeoError = layout.$.error;
+      if (!isElementVisible(nuxeoError)) {
+        await waitForAttrMutation(nuxeoError, 'hidden', null);
+      }
+      expect(isElementVisible(layout.$.container)).to.be.false;
+      expect(isElementVisible(nuxeoError)).to.be.true;
+    });
 
-  test('layout valid field invalid', async () => {
-    const layout = await fixture(html`
-      <nuxeo-layout href="${base}/dummy-layout.html" model='{"text": "valid"}'></nuxeo-layout>
-    `);
-    const validity = await layout.validate();
-    expect(validity).to.equal(false);
-  });
+    test('Should display error messages when layout is not found', async () => {
+      const layout = await fixture(
+        html`
+          <nuxeo-layout href="notfound.html"></nuxeo-layout>
+        `,
+      );
+      await flush();
+
+      const nuxeoError = layout.$.error;
+      expect(nuxeoError.code).to.equal('404');
```

Comment:
We are testing the bindings here. Don't we trust Polymer? 🙂 
WDYT about testing the rendering itself? Or, maybe migrate those tests to `nuxeo-error`?